tools/ci/testrun/script/test_open_posix/test_openposix_.py

```diff
@@ -2026,7 +2026,7 @@ def test_ltp_interfaces_sigwait_6_1(p):
 
 def test_ltp_behavior_WIFEXITED_1_2(p):
     ret = p.sendCommand(
-        "ltp_behavior_WIFEXITED_1_2", ["PASSED", "passed", "Passed", "PASS"], timeout=10
+        "ltp_behavior_WIFEXITED_1_2", ["PASSED", "passed", "Passed", "PASS"], timeout=40
```

Comment:
Extending timeout can't fix the ci failure, so we drop this change?

Reply:
Yes, I dropped two changes but kept the working one.